config/setup/markdown.go

```diff
@@ -155,6 +164,16 @@ func markdownParse(c *Controller) ([]markdown.Config, error) {
 					// only 1 argument allowed
 					return mdconfigs, c.ArgErr()
 				}
+			case "development":
+				if c.NextArg() {
+					md.Development = strings.ToLower(c.Val()) == "true"
```

Comment:
How about just `dev` instead of `development true`? It's shorter.

Better yet, what if we reverse this option so the keyword can be `cache` - if `cache` is present, the index won't be re-generated at every page load. Maybe even an optional interval argument, that could change 60s to another, custom time interval between refreshes.

Just brainstorming. Let me know what you think or would prefer.